stories/busy-indicator/busy-indicator.stories.js

```diff
@@ -68,6 +69,37 @@ export const contrastMode = () => `<div style="display:flex;justify-content:cent
 contrastMode.parameters = {
     docs: {
         iframeHeight: 200,
-        storyDescription: 'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+        storyDescription:
+            'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+    }
+};
+export const LabelledIndicator = () => `<div style="text-align: center">
```

Comment:
why do we have this inline style here and not in the scss file?

Reply:
this is retained since the the busy indicator dots has to aligned to the center of the screen.

Reply:
for extended indicator, this will need to be part of the library, because if not, it won't align to center:
<img src="https://user-images.githubusercontent.com/53509521/118593521-7a1b9380-b7c5-11eb-9d61-67324eb0d9f4.png" width="600">